diff --git a/dist/src/index.js b/dist/src/index.js
index 59b92971b78d7357fab0916ccd4a7d130d297793..0f0fa5e1bff55d5996b39680b2fe17cd64f48788 100644
--- a/dist/src/index.js
+++ b/dist/src/index.js
@@ -33,6 +33,7 @@ var import_core = __toModule(require("@miniflare/core"));
 var import_durable_objects = __toModule(require("@miniflare/durable-objects"));
 var import_html_rewriter = __toModule(require("@miniflare/html-rewriter"));
 var import_kv = __toModule(require("@miniflare/kv"));
+var import_r2 = __toModule(require("@miniflare/r2"));
 var import_runner_vm = __toModule(require("@miniflare/runner-vm"));
 var import_shared = __toModule(require("@miniflare/shared"));
 var import_sites = __toModule(require("@miniflare/sites"));
@@ -77,6 +78,7 @@ var StackedMemoryStorageFactory = class {
 var PLUGINS = {
   CorePlugin: import_core.CorePlugin,
   KVPlugin: import_kv.KVPlugin,
+  R2Plugin: import_r2.R2Plugin,
   DurableObjectsPlugin: import_durable_objects.DurableObjectsPlugin,
   CachePlugin: import_cache.CachePlugin,
   SitesPlugin: import_sites.SitesPlugin,
@@ -182,6 +184,7 @@ var MiniflareEnvironment = class {
       const state = await plugin.getObject(storage, id);
       return state.storage;
     };
+    global.mf = mf;
   }
   async teardown() {
     this.fakeTimers?.dispose();